Src/Couchbase.Linq/Couchbase.Linq.csproj

```diff
@@ -81,6 +81,7 @@
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Http.WebRequest" />
```

Comment:
I thought we were doing away with WebRequest/WebResponse in favor of HttpClient? https://issues.couchbase.com/browse/NCBC-1098